style.css

```diff
@@ -0,0 +1,72 @@
+html{
+    background-color: #ffdd99;
```

Comment:
Same color values seem to be used in multiple places. A good recommandation would be to have some variables for each used color at the top of the css file and use them accordingly. Here are some examples: https://www.w3schools.com/css/css3_variables.asp